python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -234,9 +244,21 @@ def get_allowed_deps():
                     '/usr/lib/64/libsqlite3.so.0',
                     '/usr/lib/64/libz.so.1',
                     '/usr/lib/amd64/libc.so.1',
-                    '/usr/sfw/lib/64/libcrypto.so.0.9.7',
-                    '/usr/sfw/lib/64/libssl.so.0.9.7',
                     ])
+                # OpenSSL specific deps per version.
+                import ssl
```

Comment:
Instead of these imports, I think that we could try to do a top level import like
```
from ssl import OPENSSL_VERSION_INFO
```

This will give an early failure in case ssl is not working and it make it easier to see what we use from ssl.

Reply:
Fair point, although there are plenty of other imports in the same vein, including one for `ssl`. Will correct that one too on this occasion…

I also think using `try` would make sense here and will add a warning message for versions that do not match our conditions.